lib/acme_server/account.ex

```diff
@@ -1,12 +1,19 @@
 defmodule AcmeServer.Account do
+  @type site :: String.t()
+  @type dns :: %{String.t() => String.t()}
+  @type config :: %{site: site, site_uri: URI.t(), dns: dns}
+
+  @spec create(config, String.t()) :: map()
   def create(config, client_key) do
     account = %{id: :erlang.unique_integer([:positive, :monotonic]), status: :valid, contact: []}
     AcmeServer.Db.store_new!(config, {:account, client_key}, account)
     account
   end
 
+  @spec fetch(config, String.t()) :: {:ok, any()} | :error
```

Comment:
Same thing here. With the type `t` (which describes the account) introduced, the return value becomes `{:ok, t} | :error`